tests/test_api.py

```diff
@@ -137,11 +139,29 @@ def test_get_repository(instance):
     assert repo.name == test_repo
     assert repo.owner.username == test_org
 
+def test_add_content_to_repo(instance):
+    repo = Repository.request(instance, test_org, test_repo)
+    file_content = open("tests/data/test.pcbdoc", "rb").read()
+    file_content = base64.b64encode(file_content).decode("utf-8")
+    repo.create_file("test.pcbdoc", file_content)
+    assert len(repo.get_commits()) == 2
+    assert [content.name for content in repo.get_git_content()] == ["README.md", "test.pcbdoc"]
+
+    # As an optimization, we call the generated json and svg methods so we kick off
```

Comment:
😩

Reply:
Where do we get to determine the order of the tests?

Reply:
Also i would prefer to remove this optimization. Which by the emoji I think you agree with 🤷 

Is there a noticeable difference with / without it?

Reply:
> Where do we get to determine the order of the tests?

It's top to bottom, in the order they are defined in the files.

> Also i would prefer to remove this optimization.

I removed it and refactored it to two different tests - the first testing `NotYetGeneratedException` is raised when the `get_generated_*` methods are called for the first time on a new file. I've made this change and moved the tests closer together.